### ISSUE TITLE
[Feature Request] Custom view to type byte array

### ISSUE THREAD

**Issue by hismito** [zorgiepoo/Bit-Slicer#63]
I found it's kind of hard and error-prone to 'eye-decode' a byte array to a struct.
It will be nice if a byte array can be showed as a custom struct.

**Comment by zorgiepoo** [comment]
It might be a bit easier if you use the memory viewer and toggle the data inspector, otherwise would be nice to have a parent/child grouping of multiple variables, depending on #16.

**Comment by hismito** [comment]
Yes, The data inspector did help a lot. And the grouping idea is even better than just struct. So I just close this one and keep eyes on that.
